packages/simulators/lib/simulator_manager.dart

```diff
@@ -0,0 +1,192 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io' as io;
+
+/// Creates/boots/shutsdown/lists iOS Simulators.
+///
+/// Uses `xcrun simctl` command to manage the simulators.
+///
+/// Run `xcrun simctl --help` to learn more on the details of this tool.
+class IosSimulatorManager {
```

Comment:
The constructor for this class should check that we're running on macOS and throw an error if we're not. This way, if we accidentally use this class outside macOS the error will happen earlier and will provide a good error message.

Reply:
good idea!